### ISSUE TITLE
pkg-config: ICU support is checked as required instead of optional

### ISSUE THREAD

**Issue by smancill** [arybczak/ncmpcpp#127]
The `configure` file distributed with 0.7.3 fails when boost is not compiled with ICU support.

    configure: error: Package requirements (icu-uc) were not met:

    No package 'icu-uc' found

    Consider adjusting the PKG_CONFIG_PATH environment variable if you
    installed software in a non-standard prefix.

The [`configure.ac`](https://github.com/arybczak/ncmpcpp/blob/master/configure.ac#L170-L189) file seems to have a bug when checking ICU support. The PKG_CHECK_MODULES for ICU has an empty `action-if-not-found` (746527f1ea), but this is ignored, and the default `action-if-not-found` is run, which ends the execution with an error for not having found ICU.

I have checked this on OS X. This is a problem because the distributed [ncmpcpp formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/ncmpcpp.rb) for Homebrew forces to recompile boost with ICU support, instead of using the default boost bottle.

**Comment by smancill** [comment]
I managed to fix this with this patch (OS X 10.10, pkg-config 0.29.1):

    --- configure.ac.orig	2016-01-20 10:36:22.000000000 -0300
    +++ configure.ac	2016-04-16 19:59:25.000000000 -0300
    @@ -186,7 +186,9 @@
            CPPFLAGS="$old_CPPFLAGS"
            LIBS="$old_LIBS",
        )
    -], [])
    +],
    +[]
    +)
    
    dnl =============================
    dnl = checking for boost.thread =

Now `configure` doesn't fail if boost is not compiled with ICU support.

**Comment by arybczak** [comment]
Umm, doesn't this change just insert newlines?

**Comment by arybczak** [comment]
For the record, if I make the check always fail and run it on my machine, it will just say:
```
...
checking pkg-config is at least version 0.9.0... yes
checking for ICU... yes
checking whether boost.regex was compiled with ICU support... no
checking boost/thread.hpp usability... yes
...
```

So it works.

**Comment by arybczak** [comment]
Hmm, it looks like it indeed helps. That is so stupid. Thanks for this, fixed in 0.7.4.
